src/util/string_utils.cpp

```diff
@@ -252,3 +255,23 @@ std::string wrap_line(
 
   return result;
 }
+
+char **string_to_cstr_array(const std::string &s, const char *delims)
```

Comment:
I think this should be named better - the name doesn't really imply that this is actually splitting the string... Also, given what this is used for (splitting a command string intro arguments) it's probably not complete/safe either - it doesn't handle any kind of quoting of spaces which would/might be needed for safety... e.g. if the command string is something like "mysolver --config-file '/path/with spaces/config'" I think this would break....